@here/harp-webtile-datasource/lib/WebTileLoader.ts

```diff
@@ -56,23 +55,18 @@ export class WebTileLoader extends BaseTileLoader {
                 texture.magFilter = THREE.LinearFilter;
                 texture.generateMipmaps = false;
                 this.tile.addOwnedTexture(texture);
-
-                const material = new THREE.MeshBasicMaterial({
-                    map: texture,
-                    opacity: this.dataSource.opacity,
-                    depthTest: false,
-                    depthWrite: false
-                });
-                if (this.dataSource.transparent) {
-                    enableBlending(material);
-                }
-                addGroundPlane(
+                const planeMesh = addGroundPlane(
                     this.tile,
-                    this.dataSource.renderOrder, // Remove, as `renderOrder` will be deprecated.
-                    material,
-                    true,
-                    false
+                    this.dataSource.dataSourceOrder,
```

Comment:
Hmm, I'm not sure if this is quite correct, because the dataSourceOrder is not the renderOrder, and this value is used internally as the renderOrder.

I think it would make more sense to remove the `renderOrder` from this function and just set the order to some small value.

Reply:
right, the datasourceOrder is evaluated somewhere else. I ll remove it